packaging/linux/docker/client-slim/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG SERVICE_IMAGE=keybaseio/service:latest
+
+FROM $SERVICE_IMAGE AS base
+
+FROM debian:stretch-slim
+LABEL maintainer="Keybase <admin@keybase.io>"
+
+RUN apt-get update \
+    && apt-get install -y gnupg2 procps \
+    && rm -rf /var/lib/apt/lists/*
+
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
```

Comment:
we don't need to similarly sign ours?

Reply:
That'll require your expertise, I have no idea how to approach that.